### PR TITLE
thorough review of keystone/barbican procedures

### PR DESCRIPTION
**Pull request by klgill**
NOTE: I will merge this PR after SME and peer review.

### CODE REVIEW

File: docs_user/modules/proc_adopting-key-manager-service.adoc [hunk 1]
```diff
@@ -2,34 +2,29 @@
 
 = Adopting the {key_manager}
 
-Adopting {key_manager_first_ref} means that an existing `OpenStackControlPlane` custom resource (CR), where {key_manager}
-is initialy disabled, should be patched to start the service with the configuration parameters provided by the source environment.
+To adopt the {key_manager_first_ref}, you patch an existing `OpenStackControlPlane` custom resource (CR) where {key_manager} is disabled. The patch starts the service with the configuration parameters that are provided by the {rhos_prev_long} ({OpenStackShort}) environment.
```
Comment (klgill) [inline docs_user/modules/proc_adopting-key-manager-service.adoc:5]: @xek Can you please confirm that my edits to the introduction are accurate?

File: docs_user/modules/proc_adopting-key-manager-service.adoc [hunk 2]
```diff
@@ -2,34 +2,29 @@
...
 .Procedure
 
-. Add the kek secret. In this case we are updating and using `osp-secret`,
-which contains other service passwords:
+. Add the kek secret. The following example updates `osp-secret`,
+which includes other service passwords:
 +
 ----
-oc set data secret/osp-secret "BarbicanSimpleCryptoKEK=$($CONTROLLER1_SSH "python3 -c \"import configparser; c = configparser.ConfigParser(); c.read('/var/lib/config-data/puppet-generated/barbican/etc/barbican/barbican.conf'); print(c['simple_crypto_plugin']['kek'])\"")"
+$ oc set data secret/osp-secret "BarbicanSimpleCryptoKEK=$($CONTROLLER1_SSH "python3 -c \"import configparser; c = configparser.ConfigParser(); c.read('/var/lib/config-data/puppet-generated/barbican/etc/barbican/barbican.conf'); print(c['simple_crypto_plugin']['kek'])\"")"
```
Comment (klgill) [inline docs_user/modules/proc_adopting-key-manager-service.adoc:21]: @xek Is step 1 showing an example of how to add a kek secret?

File: docs_user/modules/proc_adopting-key-manager-service.adoc [hunk 3]
```diff
@@ -2,34 +2,29 @@
...
 ----
 
-. Patch `OpenStackControlPlane` to deploy the {key_manager}:
+. Patch the `OpenStackControlPlane` CR to deploy the {key_manager}:
```
Comment (klgill) [inline docs_user/modules/proc_adopting-key-manager-service.adoc:24]: @xek In step 2, is the `openstack_control_plane.yaml` being updated?

**Comment by klgill** [comment]
@xek There's also a //TODO comment at the end of the barbican procedure. Does that still apply or can I remove it?
"//**TODO: Once different crypto plugins are supported, additional lines test those should be added."

File: docs_user/modules/proc_adopting-the-identity-service.adoc [hunk 4]
```diff
@@ -2,15 +2,14 @@
 
 = Adopting the {identity_service}
 
+To adopt the {identity_service_first_ref}, you patch an existing `OpenStackControlPlane` custom resource (CR) where the {identity_service} is disabled. The patch starts the service with the configuration parameters that are provided by the {rhos_prev_long} ({OpenStackShort}) environment.
```
Comment (klgill) [inline docs_user/modules/proc_adopting-the-identity-service.adoc:5]: @xek Is this intro accurate?

File: docs_user/modules/proc_adopting-the-identity-service.adoc [hunk 5]
```diff
@@ -2,15 +2,14 @@
...
+
 .Prerequisites
 
-* Previous Adoption steps completed. Notably,
- the xref:migrating-databases-to-mariadb-instances_migrating-databases[Migrating databases to MariaDB instances]
-must already be imported into the control plane MariaDB.
-* Ensure that you copy the fernet keys. Create the `keystone` secret, containing fernet keys:
+* Ensure that you copy the fernet keys. Create the keystone secret, containing fernet keys:
```
Comment (klgill) [inline docs_user/modules/proc_adopting-the-identity-service.adoc:9]: @xek Is the prerequisite 2 separate steps? Copy the fernet keys, then create the keystone secret containing the fernet keys?

File: docs_user/modules/proc_adopting-the-identity-service.adoc [hunk 6]
```diff
@@ -27,10 +26,10 @@ EOF
 
 .Procedure
 
-. Patch `OpenStackControlPlane` to deploy {identity_service}:
+. Patch the `OpenStackControlPlane` CR to deploy the {identity_service}:
```
Comment (klgill) [inline docs_user/modules/proc_adopting-the-identity-service.adoc:29]: @xek In step 1, is the `openstack_control_plane.yaml` file being updated?

File: docs_user/modules/proc_adopting-the-identity-service.adoc (hunk 4 above)
Comment (xek) [reply docs_user/modules/proc_adopting-the-identity-service.adoc:5]: Yep

File: docs_user/modules/proc_adopting-the-identity-service.adoc (hunk 5 above)
Comment (xek) [reply docs_user/modules/proc_adopting-the-identity-service.adoc:9]: Maybe a better wording would be:

Create the keystone secret, containing the fernet keys copied over from the source environment.

File: docs_user/modules/proc_adopting-the-identity-service.adoc (hunk 6 above)
Comment (xek) [reply docs_user/modules/proc_adopting-the-identity-service.adoc:29]: It's the CR that's being updated, so the charge is correct

File: docs_user/modules/proc_adopting-key-manager-service.adoc (hunk 3 above)
Comment (xek) [reply docs_user/modules/proc_adopting-key-manager-service.adoc:24]: It's the CR, so the change looks correct.

**Review by xek — APPROVED** [review]

File: docs_user/modules/proc_adopting-key-manager-service.adoc [hunk 7]
```diff
@@ -2,34 +2,28 @@
...
-`Keystone endpoints` should also be updated and the same crypto plugin of the source Cloud will be available. If the conditions above are met, the adoption is considered concluded.
+The {key_manager} adoption is complete if you see the following results: 
 
-[NOTE]
-This procedure configures the {key_manager} to use the `simple_crypto` backend.
-Additional backends are available, such as PKCS11 and DogTag, however they are not supported in this release.
-
-.Prerequisites
+* The `BarbicanAPI`, `BarbicanWorker`, and `BarbicanKeystoneListener` services are up and running.
+* Keystone endpoints are updated, and the same crypto plugin of the source Cloud is available.
```
Comment (gregraka) [inline docs_user/modules/proc_adopting-key-manager-service.adoc:10]: Should `Cloud` be capitalized?

**Review by gregraka — COMMENTED** [review]
Hi, Katie. Solid work. Added a comments about a few minor issues. Nothing major. Take'em or leave 'em.

File: docs_user/modules/proc_adopting-key-manager-service.adoc (hunk 7 above)
[the remark this answers is not in the document]
Comment (klgill) [reply docs_user/modules/proc_adopting-key-manager-service.adoc:10]: No, it's not a proper noun. I'll lowercase it.

**Comment by klgill** [comment]
recheck

**Comment by klgill** [comment]
@jistr or @fao89  The linting error has been happening for a few days. I've never had that error before this PR. Do you know how to resolve it?

**Comment by fao89** [comment]
there is a trailing white space after `results:`,
if you run `pre-commit run --all-files --show-diff-on-failure` it will fix it

**Pull request merged**